### ISSUE TITLE
Answers on first screen phrased as questions

### ISSUE THREAD

**Issue by msmichellegar** [joshpitzalis/shouldIWorkForFree#1]
Love it Josh! Very cute.

Just one UX thing. For the first screen, it's a bit confusing that the answers are phrased as questions. It might make more sense to ask an overarching "Who asked you to do this?" and then have the answers like "your mom", "a friend" etc. Or maybe you could ask the questions individually with "yes/no" answers.

**Comment by joshpitzalis** [comment]
good point. I wanted to keep it as close to the original as possible though. http://www.shouldiworkforfree.com/
